lark/tree.py

```diff
@@ -149,13 +148,15 @@ def iter_subtrees_topdown(self):
         Iterates over all the subtrees, return nodes in order like pretty() does.
         """
         stack = [self]
+        stack_append = stack.append
```

Comment:
I assume you added this for performance. Did you benchmark it on a realistic use-case?

Reply:
Did you try also changing `Tree` to a local variable (`_Tree` or such)?

Reply:
Not such type of thing, but I create child class of it, and enhance some part for better performance. 
Also working on multi-threading, with cythonise some part of code and logic.

Reply:
I meant specifically inside this function, to skip the global lookup.

Reply:
> Also, you're aware of https://github.com/lark-parser/lark_cython?

Yes. 
I recommend, we should start implement lark in C++ or Rust, so with WASM and  Cython (or Pybind11), we can use optimise version of it in Python and JS world.

Reply:
I'm interest to take both the tasks, if you allow.

Reply:
Sure, I will take care of it.